packages/labs/analyzer/test-files/basic-elements/src/not-lit.ts

```diff
@@ -0,0 +1,3 @@
+class LitElement {}
+
+class NotLit extends LitElement {}
```

Comment:
Getting lint errors here, we should exclude packages/labs/analyzer/test-files from linting